mocha/test/users.test.js

```diff
@@ -0,0 +1,19 @@
+const httpStatus = require("http-status");
```

Comment:
httpStatus constant isn't being used in the test. I suggest either removing this line or changing line 14 down below from '200' to 'httpStatus.OK'